src/yaml/copy-data.yaml

```diff
@@ -64,6 +65,15 @@ main:
                               prefix: "thelook_ecommerce"
                               dest_bucket_name: $${dest_tables_bucket_name}
                           result: copy_thelook_ecommerce_tables_output
+                - copy_dataplex_names_counts:
+                    steps:
+                      - copy_dataplex_names_counts_call:
+                          call: copy_objects
+                          args:
+                              source_bucket_name: $${source_bucket_name}
+                              prefix: "stage/views"
```

Comment:
Can you change "stage/views" to just "views"? I just made this change to the bucket as well.